### ISSUE TITLE
Apache Reverse Proxy with Url Prefix

### ISSUE THREAD

**Issue by ChrisShire** [seblucas/cops#340]
This is probably me being really stupid, but I wonder if COPS supports this? 

I wanted to have my COPS server URL formatted as http://mydomain.com/COPS rather than as a subdomain, eg. http://cops.mydomain.com

I'm trying to move from running calibre server where it's simple enough to specify --url-prefix /cops \ in the systemd file and use the apache proxy mod.

If COPS supports this, could anyone give me an ELI5 on how to set it up? Many thanks.

**Comment by doug-w** [comment]
I'm pretty sure everything in cops is already a relative URL and so it's
not needed.  At least I'm running two under reverse proxy and just
confirmed no special settings for them in config_local.php

**Comment by ChrisShire** [comment]
hmm, would you mind sharing your Apache configs then? (or is that so off-topic it's not allowed...?)

I have a webserver for my domain, but COPS is running on a different machine on my network and I'd really like it if I can get to COPS from http://mydomain/cops 

Help is much appreciated

**Comment by doug-w** [comment]
Ah, I see now I'm actually stripping it from the request in Nginx:

        location ~ ^/bookstech/ {
                rewrite ^/bookstech(/.*)$ $1 break;
                proxy_set_header Host $host;
                proxy_set_header X-Real-IP $remote_addr;
                proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
                proxy_pass http://192.168.86.100:9001;
        }

I think for Apache the equivilant is
    <Location /bookstech>
        ProxyPassReverse /bookstech
        SetOutputFilter INFLATE;proxy-html;DEFLATE
        ProxyHTMLURLMap /bookstech/ /
    </Location>

But I'm not exactly positive but that should be a good place to start
googling from.

**Comment by seblucas** [comment]
@ChrisShire 

I think your issue is more related to Apache Reverse Configuration than COPS. I'll close it, if you think there is something to be done in COPS, you can reopen it.
